spec/models/post_spec.rb

```diff
@@ -76,7 +76,7 @@
       Time.stub(now: Time.now)
     end
 
-    let(:post) { FactoryGirl.create(:post, created_at: 1.day.ago) }
+    let!(:post) { FactoryGirl.create(:post, created_at: 1.day.ago) }
```

Comment:
I'm not always sold on let! sharing state across tests, I keep getting bitten in subtle ways I didn't expect - I tend to prefer a ```context``` and ```before``` to describe the test setup; then do expectations about the results